.github/composer-config.json

```diff
@@ -6,6 +6,7 @@
             "symfony/http-kernel": "source",
             "symfony/messenger": "source",
             "symfony/notifier": "source",
+            "symfony/translation": "source",
```

Comment:
I added this, because my tests pass on local:
![Uploading CleanShot 2021-04-26 at 09.07.11.png…]()

Reply:
The right fix is to avoid relying on files which are inside the `Tests` folder of symfony/translation if they are meant to be exposed to other packages. Shared test cases should be in the `Test` namespace instead.

Reply:
Makes sense will move them to Test

Reply:
Hmm I removed it, but I don't rely on sth in `Tests`, and the build was red, I readded this, maybe it is because even `Translation/Test/....TestCase` is not released yet and we depend on 5.3 🤔  That at least was the case when I was dealing with Notifier bridges

Reply:
do you have a link to a failing build ? Maybe that was a case where the component-level composer installation was using a symfony/translation package from packagist rather than the patched version from your PR.